cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -466,6 +467,16 @@ instance HasSeverityAnnotation (TracePeerSelection addr) where
 
       TraceDemoteBigLedgerPeersAsynchronous {} -> Warning
 
+      TraceUseBootstrapPeersChanged {} -> Info
+      TraceBootstrapPeersFlagChangedWhilstInSensitiveState -> Info
+
+      TraceLedgerStateJudgementChanged {} -> Info
+      TraceOnlyBootstrapPeers {}          -> Info
+
+      TraceOutboundGovernorCriticalFailure {} -> Error
+
+      TraceDebugState {} -> Info
```

Comment:
That's for sure ought to be `Debug` level:
```suggestion
      TraceDebugState {} -> Debug
```

Reply:
Nope. In spite of the name these log messages are manually triggered.
Having them at `Debug` means that no SPO will ever have them enabled when we want the data for troobleshooting.